fitter/probabilities/pulsars.py

```diff
@@ -249,17 +249,24 @@ def cluster_component(model, R, mass_bin, DM=None, ΔDM=None, DM_mdata=None, *,
 
         else:
             # This is the case where our probability distribution doesn't
-            # integrate to one, just don't cut anything off, log the
-            # normalization and manually normalize it.
+            # integrate to one.
+
+            # If the area is way less than 1, we should just throw an exception
+            if norm < 0.9:
+                logging.error(
```

Comment:
Do we need to be both logging and raising the exception? Wasn't totally sure what you'd prefer but this feels redundant.